pyproject.toml

```diff
@@ -63,6 +63,7 @@ apache-beam = "^2.48.0"
 pillow = "^9.5.0"
 cairosvg = "^2.7.0"
 colorama = "^0.4.6"
+tensorflow = "^2.12.0"
```

Comment:
this should definitely be an optional dep 👀  check how the beam dependency is defined in the `[tool.poetry.extras]` section (and we should probably add a check to see if tf is installed before running any code that uses it)

Reply:
*Done x 2